### DISCUSSION TITLE
How is data migration handled?

### DISCUSSION THREAD

**Discussion by vin47** [FerretDB/FerretDB#270] [General]
If we are to migrate from mongo db to ferret db, how is data migration handled?

**Comment by AlekSi** [comment]
In the Alpha stage, we plan to support only mongodump and mongorestore. In fact, they might already be supported but are not actively tested. I created an issue to track that: https://github.com/FerretDB/FerretDB/issues/271

In a later stage, we might add support for the MongoDB replication protocol, so FerretDB can be added to the MongoDB replica set. There is no time estimation for that yet.

**Reply by vin47** [reply]
Thanks @AlekSi !!
